### PR TITLE
Added a Redis System + Documentation

### PR DESCRIPTION
**Pull request by itay64**
Added a Redis system:
- Publish Messages
- Listen To Incoming Messages
- Added ENV Support (Unsure if it works lmao) (Can also use the 'redis()' function)
- Added documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -426,6 +426,56 @@ NameCacheService.uuidFromName("stxphen")
 
 Currently the only way to configure your MongoDB "cache" for UUIDs and names, is to have an Environment variable called `NAME_CACHE_COLLECTION` with the value being what you want to call the collection. Don't want to use the Mongo cache? Disable `useMongoCache` in the settings. 
 
+# Redis
+Twilight has a Redis system that lets you publish messages and listen to incoming messages on any channel you'd like.
+
+#### Environment variables
+You can use the following Environment variables for your MongoDB:
```
Comment (kotlinsyntax) [inline README.md:433]: This should be Redis, Not Mongo

Comment (itay64) [reply README.md]: Changed

Comment (kotlinsyntax) [reply README.md]: lgtm

Comment (itay64) [reply README.md]: I updated the readme

Comment (itay64) [reply README.md]: is there anything else?

**Review by joshbker — APPROVED** [review]

**Pull request merged**